sda-commons-server-s3/src/test/java/org/sdase/commons/server/s3/test/TestApp.java

```diff
@@ -14,7 +14,6 @@
       S3Bundle.builder()
           .withConfigurationProvider(Config::getS3Config)
           .withTracer(mockTracer)
-          .withHealthCheck("testbucket")
```

Comment:
Why did you remove the healthcheck for the test app?

Reply:
The test using it tests the tracing. I created new test apps for the different health check scenarios.